README.md

```
@@ -1,2 +1,89 @@
+- [x] Directional lights
+- [x] Ambient lights
+- [x] Point lights
+- [x] Diffuse lights
+- [x] Colored lights
+- [x] Multiple point lights
+- [x] Drop shadows
+
+#### Primitives
+
+- [x] Spheres
+- [x] Planes 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)